test/fixtures/dialects/oracle/comment.sql

```diff
@@ -0,0 +1,2 @@
+COMMENT ON COLUMN employees.job_id 
+   IS 'abbreviated job title';
```

Comment:
Currently we only have a test case for `COMMENT ON COLUMN` here?

We should also have tests below for TABLE, INDEXTYPE, OPERATOR, and MATERIALIZED VIEW.

(n.b. as mentioned in other review comments, I think some of these other possible comment targets might not parse. Always best to have all cases covered in tests to prove that a PR works)

Reply:
Yep, good point... easy coverage increase